smartsupp.php

```diff
@@ -347,6 +347,12 @@ public function hookDisplayBackOfficeHeader()
         $js = '';
 
         if (strcmp(Tools::getValue('configure'), $this->name) === 0) {
+            \Media::addJsDef([
```

Comment:
What about the other hook where smartsupp.js file is appended? not sure when it loads

Reply:
used for different PS versions. backOfficeHeader (1.6.x), displayBackOfficeHeader > 1.6